README.md

```diff
@@ -26,7 +26,11 @@ The following example shows the container used for running Renovate:
       target: swissgrc/azure-pipelines-renovate:latest
 ```
 
-### Tags
+## Included Software
+- [swissgrc/azure-pipelines-azurecli:net8](https://github.com/swissgrc/docker-azure-pipelines-azurecli-net8) as base image
```

Comment:
Uses our Node.js image, not the Azure CLI

https://github.com/swissgrc/docker-azure-pipelines-renovate/blob/9b8d1d44f92bebb4ea0f6e7bb2922d3b0ddce414/Dockerfile#L1